includes/blocks.inc

```diff
@@ -556,13 +555,20 @@ function islandora_solr_advanced_search_form($form, &$form_state) {
       'terms' => array(''),
     );
   }
+
   $terms = array(
     '#type' => 'markup',
     '#prefix' => '<div id="islandora-solr-advanced-terms">',
     '#suffix' => '</div>',
     '#tree' => TRUE,
   );
   foreach ($values['terms'] as $i => $value) {
+    if (variable_get('islandora_solr_advanced_search_block_lucene_syntax_escape', FALSE)) {
```

Comment:
Given we're replacing slashes independent of this... should probably be restoring independently?